mail_edit/static/src/js/mail_edit.js

```diff
@@ -0,0 +1,47 @@
+/* © 2014-2017 Sunflower IT <www.sunflowerweb.nl>*/
+
+"use strict";
+openerp.mail_edit = function (instance) {
+    var _t = instance.web._t;
+    instance.mail.ThreadMessage.include({
+        bind_events: function () {
+            this._super.apply(this, arguments);
+            this.$('.oe_edit').on('click', this.on_message_edit);
+        },
+
+        on_message_edit: function () {
+            var context = {};
+
+            // this.id has the widget id
+            // this.model has the model name or id
+            // this.res_id has the record id
+
+            // save the widget object in a var.
+            var self = this;
+
+            // Get the action data and execute it to open the composer wizard
+            var do_action = this.do_action;
+
+            this.rpc("/web/action/load", {
```

Comment:
stick to the habit of always returning deferred object you create, then overrides can so something with it